src/Renci.SshNet/Security/KeyExchange.cs

```diff
@@ -183,10 +254,6 @@ public Cipher CreateServerCipher()
 
             serverKey = GenerateSessionKey(SharedKey, ExchangeHash, serverKey, _serverCipherInfo.KeySize / 8);
 
-            DiagnosticAbstraction.Log(string.Format("[{0}] Creating {1} server cipher.",
```

Comment:
I'd at least like to discuss why you prefer to remove these. The logging you added in this class does not help if you have multiple key exchanges in progress. The previous logging used a session id prefix for this purpose.

I'd prefer to use that prefix whether it's accessible.

Reply:
I wanted the "offer" messages and figured I could put these ones in the same place. I'll put them back